public/jsx/app.jsx

```diff
@@ -50,7 +58,7 @@ module.exports = {
   init: function() {
     Router.run(routes, Router.HashLocation, function(Root) {
       // Doesn't actually render anything.
-      React.render(<Root/>, document.body);
+      React.render(<Root/>, document.getElementById('view'));
```

Comment:
don't you want to continue rendering to the ``body`` now that all the HTML is there? Otherwise it seems like you might have multiple ``<main>`` tags